components/collections/addCollection.vue

```diff
@@ -78,6 +78,7 @@ export default {
     },
     hideModal() {
       this.$emit("hide-modal")
+      this.$data.name = undefined
```

Comment:
@myussufz can you tell me what does this LOC do?

Reply:
I realize, every time I add a new collection, the input field keep the state, so I make it empty, every time the modal is hidden